README.md

```diff
@@ -64,316 +55,124 @@ machine during metric analysis.
-contains its own Python runtime, but has an external depencency on the
-[Visual C++ Compiler for Python 2.7](https://www.microsoft.com/EN-US/DOWNLOAD/DETAILS.ASPX?ID=44266)
-in order to operate.  The agent supports Windows Server 2008 and above.
+### __Hello.__
+
+### __Get started with Smart Agent using the 3 steps below.__
+
+
+### Step 1. Install SignalFx Smart Agent on Single Host 
+
+_Note: Uninstall any existing collectd agent before installing SignalFx Smart Agent._
 
-To get started deploying the Smart Agent directly on a host, see the
-[Smart Agent Quickstart](./docs/smart-agent-quickstart.md) guide.
+__Linux:__ The agent is available for Linux in both a containerized and standalone form. In both forms the dependencies are completely bundled along with the agent, including a Java JRE runtime and a Python runtime, so there are no additional dependencies required. The agent works on any modern Linux distribution (kernel version 2.6+).
 
-### Deployment
-We support the following deployment/configuration management tools to automate the
-installation process.  See [Bundles](#bundles) for a list of underlying
-packages for the agent.
+Get your API_TOKEN from: /etc/signalfx/token 
 
-#### Installer Script
-For non-containerized environments, there is a convenience script that you can
-run on your host to install the agent package.  This is useful for testing and
-trials, but for full-scale deployments you will probably want to use a
-configuration management system like Chef or Puppet.
+Determine YOUR\_SIGNAL\_FX_REALM from: your profile page in the SignalFx web application.
 
-##### Linux
-You can [view the source for the installer script](./deployments/installer/install.sh)
-and use it on your hosts by running:
+To install the Smart Agent on a single Linux host, enter:
 
 ```sh
 curl -sSL https://dl.signalfx.com/signalfx-agent.sh > /tmp/signalfx-agent.sh
-sudo sh /tmp/signalfx-agent.sh YOUR_SIGNALFX_API_TOKEN
-```
+sudo sh /tmp/signalfx-agent.sh --realm YOUR_SIGNALFX_REALM YOUR_SIGNALFX_API_TOKEN
+````
+
+__Windows:__ The agent is also available on Windows in standalone form.
 
-##### Windows
-The Agent has two dependencies on Windows which must be satisfied before running the installer script.
+Ensure that the following dependencies are installed:
 
-- [.Net Framework 3.5](https://docs.microsoft.com/en-us/dotnet/framework/install/dotnet-35-windows-10) (Windows 8+)
-- [Visual C++ Compiler for Python 2.7](https://www.microsoft.com/EN-US/DOWNLOAD/DETAILS.ASPX?ID=44266)
+[.Net Framework 3.5] (https://docs.microsoft.com/en-us/dotnet/framework/install/dotnet-35-windows-10) (Windows 8+)
+
+[Visual C++ Compiler for Python 2.7](https://www.microsoft.com/EN-US/DOWNLOAD/DETAILS.ASPX?ID=44266)
 
-The installer script is written for Powershell v3.0 and above and will not function correctly on earlier versions.
+* Get your API\_TOKEN from:  __\ProgramData\SignalFxAgent\token or the Organization->Access Token__ tab in the SignalFx application.
```

Comment:
See? This info is different from the info above about how to get your token. We should come up with a single recommendation.

Reply:
ok. **Organization Settings => Access Token** method.